src/Appwrite/Platform/Modules/Functions/Workers/Builds.php

```diff
@@ -713,6 +720,107 @@ protected function buildDeployment(Device $deviceForFunctions, Func $queueForFun
 
             Console::success("Build id: $buildId created");
 
+            if ($resource->getCollection() === 'sites') {
+                try {
+                    $rule = Authorization::skip(fn () => $dbForPlatform->findOne('rules', [
+                        Query::equal("projectInternalId", [$project->getInternalId()]),
+                        Query::equal("resourceType", ["deployment"]),
+                        Query::equal("resourceInternalId", [$deployment->getInternalId()])
+                    ]));
+
+                    if ($rule->isEmpty()) {
+                        throw new \Exception("Rule for build not found");
+                    }
+
+                    $client = new FetchClient();
+                    $client->addHeader('content-type', FetchClient::CONTENT_TYPE_APPLICATION_JSON);
+
+                    $bucket = $dbForPlatform->getDocument('buckets', 'screenshots');
+
+                    $configs = [
+                        'screenshot' => [
+                            'headers' => [ 'x-appwrite-hostname' => $rule->getAttribute('domain') ],
+                            'url' => 'http://traefik/?appwrite-preview=1&appwrite-theme=light',
```

Comment:
Would it remain as traefik? Should we use an env var instead of hardcoding?

Reply:
It's internal complxity, it should never be changed by instance of server on which its running, so I would say it's fine to hard-code. It's a hostname that appwirte-browser can use to talk to Appwrite API. Those names wont change unless we decide to remove Traefik, which would be pretty rare. Tests would also fail so indication would be good enough to update it here

Reply:
Would it behave the same on k8s as on swarm/compose?

Reply:
I checked what migration is doing, its doing `appwrite`. So switching from `traefik` to `appwrite`, and keeping it hard-coded.